docs/pages/router/ui/tabs.mdx

```diff
@@ -35,25 +40,25 @@ Expo Router offers a set of components to create custom Tabs layouts. Unlike the
 
 ## Creating routes
 
-The `<TabList />` component is the "holding" component for routes. `<TabTrigger />`'s defined as a child of the `<TabList />` specific what routes are present as a tab. A `<TabTrigger />` within a `<TabList />` must include a `name` and a `href` prop. `<TabList />` needs to be an immediate child of `<Tabs />`
+The `TabList` component is the "holding" component for routes. `TabTrigger`'s defined as a child of the `TabList` specific what routes are present as a tab. A `TabTrigger` within a `TabList` must include a `name` and a `href` prop. `TabList` needs to be an immediate child of `Tabs`.
 
-`<TabList />` is both the configuration and default appearance of the `<Tabs />`. Triggers present within the `<TabList />` dictate what routes are possible within the `<Tabs />`
+`TabList` is both the configuration and default appearance of the `Tabs`. Triggers present within the `TabList` dictate what routes are possible within the `Tabs`.
 
-> `name` can be any `string`. This is a user-defined name for the Tab.
+> **Note:** `name` can be any `string`. This is a user-defined name for the Tab.
 
 ### Dynamic routes
 
-Dynamic routes are allowed and can be provided with values via the `href`
+Dynamic routes are allowed and can be provided with values via the `href`.
 
 <FileTree files={['_layout.tsx', ['[slug].tsx']]} />
 
-The trigger `<Trigger name="dynamic page" href="/hello-world" />` will create a tab for `[slug].tsx` with the params `{ slug: 'hello-world' }`
+The trigger `<Trigger name="dynamic page" href="/hello-world" />` will create a tab for **[slug].tsx** with the params `{ slug: 'hello-world' }`.
 
 ### Ambagious routes
 
 <FileTree files={['_layout.tsx', ['(one,two)/route.tsx', 'A route within a shared group']]} />
 
-The `href` values provided to `<TabTrigger />` cannot be ambagious and must always point to a single route. The href `/route` is not allowed, as it could refer to either `/(one)/route` or `/(two)/route`.
+The `href` values provided to `<abTrigger` cannot be ambagious and must always point to a single route. The href `/route` is not allowed, as it could refer to either `/(one)/route` or `/(two)/route`.
```

Comment:
`TabTrigger` instead of `<abTrigger`, right?

Reply:
Good catch 😅